lib/src/models/project_model.dart

```diff
@@ -40,16 +40,15 @@ class Project {
     final configFile = _fvmConfigPath(path);
     final legacyConfigFile = _legacyFvmConfigPath(path);
 
+    final config = ProjectConfig.loadFromPath(configFile);
+
     // Used for migration of config files
     final legacyConfig = ProjectConfig.loadFromPath(legacyConfigFile);
 
-    if (legacyConfig != null) {
+    if (config == null && legacyConfig != null) {
       legacyConfig.save(configFile);
```

Comment:
Does this need to load the new config?

Reply:
Yes, this is the migration of the new config that does not exist. It creates a new one based on the old one.

Reply:
Yes, but `config` is still null right, doesn't it need to be loaded after migration?